crane/container.go

```diff
@@ -788,11 +790,13 @@ func (c *container) Unpause() {
 // Remove container
 func (c *container) Rm(force bool) {
 	if c.Exists() {
-		if !force && c.Running() {
+		containerIsRunning := c.Running()
```

Comment:
I thought that too, but at that time it is still running. I don't think there is a name for it that really fits :(